### PR TITLE
Feature/dwo init command database setup

### PR DESCRIPTION
**Pull request by larcin**

### CODE REVIEW

File: src/commands/init.js [hunk 1]
```diff
@@ -24,11 +40,38 @@ function handleInitError(rawError) {
   }
 }
 
+async function handleDatabaseConfiguration() {
+  if (process.env.DATABASE_URL) {
+    return null;
+  }
+
+  logger.success('✅ Checking your database setup');
```
Comment (adriguy) [inline src/commands/init.js:48]: If we put your step in a spinner too, we won't need this log.
Let's talk about it to make all step success messages homogeneous.
image

File: src/commands/init.js [hunk 2]
```diff
@@ -24,11 +40,38 @@ function handleInitError(rawError) {
   }
 }
 
+async function handleDatabaseConfiguration() {
+  if (process.env.DATABASE_URL) {
+    return null;
+  }
```
Comment (adriguy) [inline src/commands/init.js:46]: I think we want the success message "✅Checking your database setup" even when we figured out that there was already a DATABASE_URL set up.

File: src/services/prompter/database-prompter.js [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+    this.handleSchema();
+    this.handleHostname();
+    this.handlePort();
+    this.handleUser();
+    this.handlePassword();
+    this.handleSsl();
+    this.handleMongodbSrv();
+  }
+
+  // NOTICE: This is not used but required to stay iso with lumber for now
```
Comment (adriguy) [inline src/services/prompter/database-prompter.js:33]: 👏

File: src/commands/init.js (hunk 1 above)
Comment (larcin) [reply src/commands/init.js:48]: As said in private, this will be handle in next PR about spinner

**Pull request merged**